pkgs/top-level/python-packages.nix

```diff
@@ -2616,17 +2623,29 @@ in modules // {
       py.test $out/${python.sitePackages}
     '';
 
-    # Marked broken since according to test
+    # FIXME: make gdk_pixbuf dependency optional (as wel as xcfffi)
     # Happens with 0.7.1 and 0.7.2
     # OSError: dlopen() failed to load a library: gdk_pixbuf-2.0 / gdk_pixbuf-2.0-0
 
-    patchPhase = ''
+    patches = [
```

Comment:
On Sun, Feb 07, 2016 at 05:18:32AM -0800, Nikolay Amiantov wrote:

> >      # Happens with 0.7.1 and 0.7.2
> >      # OSError: dlopen() failed to load a library: gdk_pixbuf-2.0 / gdk_pixbuf-2.0-0
> >  
> > -    patchPhase = ''
> > +    patches = [
> 
> How had you found out this sha256 checksum? I ask because `nix-prefetch-url` cannot be used for `fetchpatch`.
> 

Then we found source of failure ;)
Now I need to know, how to pick proper hash for fetchpatch.